index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="https://fonts.googleapis.com/css?family=Crete+Round:regular,italic" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" rel="stylesheet" />
+
+    <link rel="stylesheet" href="./styles.css">
+
+    <script src="https://kit.fontawesome.com/e971436820.js" crossorigin="anonymous"></script>
+
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <nav class="nav-bar">
+            <h3>John Doe</h3>
+            <i class="fas fa-bars fa-lg"></i>
+        </nav>
+        <section class="intro">
+            <header>
+                <p>Hey There.<br>I’m Alex</p>
+            </header>
+            <div class="primary-content">
+                <h3>I’am a Software Developer</h3>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+            </div>
+            <div class="socials">
+                <a href=""><i class="fab fa-github fa-2x"></i></a>
+                <a href=""><i class="fab fa-linkedin-in fa-2x"></i></a>
+                <a href=""><i class="fab fa-angellist fa-2x"></i></a>
+                <a href=""><i class="fab fa-twitter fa-2x"></i></a>
+                <a href=""><i class="fab fa-medium-m fa-2x"></i></a>
+            </div>
```

Comment:
[Optional]
You implemented the social media icons in a good way. Let us make them look better by reducing their size so they will match the Figma design.